snapcraft/internal/pluginhandler/__init__.py

```diff
@@ -569,8 +570,24 @@ def mark_build_done(self):
             scriptlet_metadata=self._scriptlet_metadata[steps.BUILD]))
 
     def _get_machine_manifest(self):
+        # Use subprocess directly here. common.run_output will use binaries out
+        # of the snap, and we want to use the one on the host.
+        try:
+            output = subprocess.check_output(['uname', '-srvmpio'])
```

Comment:
mind using uname's long options for clarity and purpose

Reply:
Sure, done.